<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>450</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>450</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>450</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>фывфыв</string>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="shedule">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>320</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Показать расписание</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>295</y>
      <width>241</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Пароль администратора</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>328</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Serif</family>
     </font>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="admin">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>370</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Вход</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radio_sr">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>250</y>
      <width>82</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Среда</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radio_vt">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>230</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Вторник</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radio_pn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>210</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Понедельник</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_pic">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-22</y>
      <width>800</width>
      <height>471</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="cancel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>380</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Отменить запись</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radio_pt">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>290</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Пятница</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radio_cht">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>270</y>
      <width>82</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Четверг</string>
    </property>
   </widget>
   <zorder>label_pic</zorder>
   <zorder>shedule</zorder>
   <zorder>label_3</zorder>
   <zorder>lineEdit_2</zorder>
   <zorder>admin</zorder>
   <zorder>radio_sr</zorder>
   <zorder>radio_vt</zorder>
   <zorder>radio_pn</zorder>
   <zorder>cancel</zorder>
   <zorder>radio_pt</zorder>
   <zorder>radio_cht</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
